internal/config/config.go

```diff
@@ -71,6 +71,12 @@ type (
 		Pprof PprofDebug `yaml:"pprof"`
 	}
 
+	Kube struct {
+		Enabled *bool  `yaml:"enabled"`
+		Config  string `yaml:"config"`
+		Node    string `yaml:"nodeName"`
```

Comment:
```suggestion
		NodeName    string
```

Storing the NodeName in config yaml is a double edged sword .. It implies that all keplers on all nodes will watch the Node in the config.yaml unless --kube.node is passed.

Reply:
`--kube.node-name` is mandatory in kepler